cinema_service/settings.py

```diff
@@ -136,3 +136,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+DJANGO_SETTINGS_MODUL = "cinema_service.settings"
```

Comment:
There is a typo in the `DJANGO_SETTINGS_MODUL` variable. It should be `DJANGO_SETTINGS_MODULE`.